testing/workflows/components/workflows.libsonnet

```diff
@@ -211,12 +211,23 @@
                           "",
                     ],
                   },
-
                   {
                     name: "pytorchjob-deploy",
                     template: "pytorchjob-deploy",
                     dependencies: ["deploy-kubeflow"],
                   },
+                  // Don't run argo test for gke since
+                  // it runs in the same cluster as the
+                  // test cluster. For minikube, we have
```

Comment:
Isn't it deployed in its own namespace, so it's okay to be in the same gke cluster?

Reply:
Having multiple controllers for argo in the same cluster causes race conditions for argo workflows - we had a downtime in the test infrastructure because of this last week.

Reply:
I see